app/templates/home.html

```diff
@@ -73,7 +73,7 @@ <h1><i class="calendar icon"></i>{{ _("Meal planning") }}</h1>
       </td>
       <td></td>
       <td>
-        <a class="ui labeled icon button" href="{{ url_for('recipes', schedule='24-3-2020') }}">
+        <a class="ui labeled icon button" href="{{ url_for('main.recipes', schedule='24-3-2020') }}">
```

Comment:
each of these changes is related to using blueprint here:
https://github.com/juriansluiman/groceri.es/pull/30/files#diff-0c2bad94cd303c5944a38a94f62a63355b8bed2e102dc87c0f44de95cf135b8aR26